src/plugins/kibana_react/public/markdown/markdown.tsx

```diff
@@ -78,29 +78,31 @@ export interface MarkdownProps extends React.HTMLAttributes<HTMLDivElement> {
   markdown?: string;
   openLinksInNewTab?: boolean;
   whiteListedRules?: string[];
+  onRender?: () => void;
 }
 
-export class Markdown extends PureComponent<MarkdownProps> {
-  render() {
-    const { className, markdown = '', openLinksInNewTab, whiteListedRules, ...rest } = this.props;
+export const Markdown = (props: MarkdownProps) => {
+  useEffect(() => {
+    props.onRender?.();
+  }, [props]);
```

Comment:
Is it meant to be called on every prop change? If not we probably only want to declare `onRender` inside the dependencies array.

Reply:
@sebelga right, it should be executed on each prop change.